nohup: ignoring input
Importing libraries
/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/typing.py:72: UserWarning: An issue occurred while importing 'torch-scatter'. Disabling its usage. Stacktrace: /lustre/hpc/icecube/cyan/.local/lib/python3.9/site-packages/torch_scatter/_version_cuda.so: undefined symbol: _ZN5torch3jit17parseSchemaOrNameERKSs
  warnings.warn(f"An issue occurred while importing 'torch-scatter'. "
/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/typing.py:83: UserWarning: An issue occurred while importing 'torch-cluster'. Disabling its usage. Stacktrace: /lustre/hpc/icecube/cyan/.local/lib/python3.9/site-packages/torch_cluster/_version_cuda.so: undefined symbol: _ZN5torch3jit17parseSchemaOrNameERKSs
  warnings.warn(f"An issue occurred while importing 'torch-cluster'. "
/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/typing.py:110: UserWarning: An issue occurred while importing 'torch-sparse'. Disabling its usage. Stacktrace: /lustre/hpc/icecube/cyan/.local/lib/python3.9/site-packages/torch_sparse/_version_cuda.so: undefined symbol: _ZN5torch3jit17parseSchemaOrNameERKSs
  warnings.warn(f"An issue occurred while importing 'torch-sparse'. "
Traceback (most recent call last):
  File "/lustre/hpc/icecube/cyan/IceCube_project_share/icecube-projects/luc/training_script.py", line 16, in <module>
    from dataset_multifile_flavours import PMTfiedDatasetPyArrow
  File "/lustre/hpc/icecube/cyan/IceCube_project_share/icecube-projects/luc/dataset_multifile_flavours.py", line 6, in <module>
    from torch_geometric.data import Data
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/__init__.py", line 13, in <module>
    import torch_geometric.datasets
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/datasets/__init__.py", line 101, in <module>
    from .explainer_dataset import ExplainerDataset
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/datasets/explainer_dataset.py", line 9, in <module>
    from torch_geometric.explain import Explanation
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/explain/__init__.py", line 3, in <module>
    from .algorithm import *  # noqa
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/explain/algorithm/__init__.py", line 1, in <module>
    from .base import ExplainerAlgorithm
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/explain/algorithm/base.py", line 14, in <module>
    from torch_geometric.nn import MessagePassing
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/nn/__init__.py", line 5, in <module>
    from .to_hetero_with_bases_transformer import to_hetero_with_bases
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/nn/to_hetero_with_bases_transformer.py", line 9, in <module>
    from torch_geometric.nn.conv import MessagePassing
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/nn/conv/__init__.py", line 8, in <module>
    from .gravnet_conv import GravNetConv
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_geometric/nn/conv/gravnet_conv.py", line 13, in <module>
    from torch_cluster import knn
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch_cluster/__init__.py", line 18, in <module>
    torch.ops.load_library(spec.origin)
  File "/groups/icecube/cyan/.local/lib/python3.9/site-packages/torch/_ops.py", line 1295, in load_library
    ctypes.CDLL(path)
  File "/usr/lib64/python3.9/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /lustre/hpc/icecube/cyan/.local/lib/python3.9/site-packages/torch_cluster/_version_cuda.so: undefined symbol: _ZN5torch3jit17parseSchemaOrNameERKSs
